README.md

```diff
@@ -92,7 +92,9 @@ With [Lazy.nvim](https://github.com/folke/lazy.nvim):
   config = function(_, opts)
     require("emoji").setup(opts)
     -- optional for telescope integration
-    require("telescope").load_extension("emoji")
+    local ts = require('telescope').load_extension 'emoji'
+    -- optional keymap for existing kickstart.nvim users
```

Comment:
This does not solely apply to kickstart.nvim users. `vim.keymap.set` is a legit command in every Neovim (distribution). Please remove the "kickstart.nvim" hint.